### ISSUE TITLE
Moveit_python pickup function

### ISSUE THREAD

**Issue by pet1330** [pet1330/FInCoR#25]
I have been setting up Moveit to work correctly on Baxter,  this includes the following:
- [x] Create a node to publish all collision objects in the environment such as tables, kinects and walls
- [x] Create virtually Baxter environment for objects around Baxter 
- [x] Allow planning for each arm, including collision avoidance
- [x] Get both arms executing simultaneously 
- [x] Allow pose target to be set for end effectors (You can provide an xyz position or a pose msg) 
- [ ] Convert this to an action server 
- [ ] Look into pickup functions from Moveit_python module ( @marc-hanheide: I'm struggling to get this to work, is it worth pursuing, or should I just implement my own? )
- [ ] Implement own pickup function, think about action server in #19 
- [ ] Create simple task to sort blocks into order (simple demo)

**Comment by marc-hanheide** [comment]
👍  re the pick up functions: I don't know them. What do the offer? Anything useful? And what's the problem?

**Comment by pet1330** [comment]
**TL;DR Could not get pick and place to work so I am making my own basic version...**

The issue is that the default grip position is to grab the object top down, however, this is being [flips somewhere](https://groups.google.com/a/rethinkrobotics.com/forum/#!topic/brr-users/qfHbVjJIMqg) causing the default grip position to be from the bottom of the object. This causes the planning stage to fail as there is no way of grabbing an object when the bottom when it is sitting on a table.

However, when the object is placed in a position that it can be reached from the bottom, or when the object is turned upside down, it complains that the _"right_gripper_base" is not known to model 'baxter'_ which also causing the IK to fail

I have check the urdf files and they do not have have the right_gripper_base, but the /tf tree does. However, when I tried to alter this, it caused RVIZ to not display correctly and still left the original issue. So this might be the right direction, but the wrong file, I'm just not sure. All of the [issues on the forums](https://groups.google.com/forum/#!topic/moveit-users/gPJtzIX68yw) are from 2014 and using Hydro, but I cannot find a more up to date source.

As I've already spent more time trying to solve this that I'd have liked, I'm just going to implement a basic top down pick_up action server which just plans to a position just above the object, opens the grippers, avoidance, moves down to the object's position, closes the grippers.

**Comment by pet1330** [comment]
Change of plan, this is no longer happening
